core/src/main/java/tc/oc/pgm/api/Permissions.java

```diff
@@ -37,6 +37,7 @@
   String BAN = ROOT + ".ban"; // Access to the /ban command
   String FREEZE = ROOT + ".freeze"; // Access to the /freeze command
   String VANISH = ROOT + ".vanish"; // Access to /vanish command
+  String MAPDEV = ROOT + ".mapdev"; // Access to mapdev related commands
```

Comment:
You should probably add this permission to `Permission DEVELOPER`. Also to `registerAll()`

Reply:
Good catch, will do 👍